lighthouse-cli/bin.js

```diff
@@ -94,6 +94,12 @@ async function begin() {
     cliFlags.outputPath = 'stdout';
   }
 
+  // show deprecation message for disableDeviceEmulation
```

Comment:
maybe add a note we should remove for v6? (we don't have to remove it then, but then we'll know it's alright to at any time after that :)